docs/resources/Channel.md

```diff
@@ -555,14 +556,15 @@ A thread member is used to indicate whether a user has joined a thread or not.
 
 Embed types are "loosely defined" and, for the most part, are not used by our clients for rendering. Embed attributes power what is rendered. Embed types should be considered deprecated and might be removed in a future API version.
 
-| Type    | Description                                        |
-|---------|----------------------------------------------------|
-| rich    | generic embed rendered from embed attributes       |
-| image   | image embed                                        |
-| video   | video embed                                        |
-| gifv    | animated gif image embed rendered as a video embed |
-| article | article embed                                      |
-| link    | link embed                                         |
+| Type                       | Description                                        |
+|----------------------------|----------------------------------------------------|
+| rich                       | generic embed rendered from embed attributes       |
+| image                      | image embed                                        |
+| video                      | video embed                                        |
+| gifv                       | animated gif image embed rendered as a video embed |
+| article                    | article embed                                      |
+| link                       | link embed                                         |
+| auto_moderation_message    | embed containing blocked content by automod        |
```

Comment:
I think "automod alert embed" is probably better, since you can flag without blocking, and that sends teh same type of embed

Reply:
It's also more apt given the format of the descriptions above it

Reply:
Done!